.github/workflows/ubuntu_tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           - 3.7.10
           - 3.8.10
           - 3.9.5
-          - 3.10.0rc2
+          - 3.10.0
```

Comment:
This will probably need to wait for a release at  https://github.com/actions/python-versions/releases but those tend to happen very quickly.

Reply:
Actually, as far as i understand it, those are only used for installing python using pyenv itself, so it shouldn't need a release from actions/python-version.